src/common/transformations/include/ngraph_ops/type_relaxed.hpp

```diff
@@ -323,6 +323,10 @@ std::shared_ptr<Node> TypeRelaxed<BaseOp>::clone_with_new_inputs(const OutputVec
     }
 
     new_node->validate_and_infer_types();
+
+    // clean rt_info and friendly name (that mustn't be cloned due to semantic of this method)
+    new_node->set_friendly_name("");
```

Comment:
Why not: new_node->set_friendly_name(this->get_friendly_name());?

Reply:
Because we should remove friendly name duplication in this method (to align behavior with other `clone_with_new_inputs` methods and avoid some issues that might appear due to two nodes with the same friendly name in the model.